src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -77,6 +77,16 @@ public PersonBuilder(Person personToCopy) {
         medCons = new HashSet<>(personToCopy.getMedCons());
     }
 
+    /**
+     * Resets the parameters of properties that cannot be added to {@code Person}.
+     */
+    public PersonBuilder withoutExtraProperties() {
+        this.medCons = new HashSet<>();
+        this.priority = new Priority("NONE");
```

Comment:
Call default constructor for "NONE" constructor has been overloaded.

`this.priority = new Priority();`